x-pack/plugins/ml/public/components/field_data_card/_field_data_card.scss

```diff
@@ -68,6 +68,11 @@
     display: inline-block;
   }
 
+  .stat.min.value, .stat.max.value, .stat.median.value {
+    font-size: $euiFontSizeS;
+    @include euiTextTruncate;
```

Comment:
I'm not seeing the ellipsis on IE or FF, although it is getting truncated OK:

![image](https://user-images.githubusercontent.com/7405507/58331758-bcf23d00-7e31-11e9-8dcc-a977cbda1f57.png)

Reply:
Fixed in 7e4fd42721475403e535c3af3983859695e680b2